modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,213 @@
+* Executes the `configure-ovs.sh` shell script that references the new interface definition.
+* Adds `br-ex` and `br-ex1` to the host node.
+* Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
+
+For more information about useful situations for the additional `br-ex` bridge and a situation that always requires the default `br-ex` bridge, see "Configuration for a localnet topology".
+
+.Procedure
+
+. Optional: Create an interface connection that your secondary bridge, `br-ex1`, can use by completing the following steps. The example steps show the creation of a new bond and its dependent interfaces that are all defined in a machine configuration manifest file. The additional bridge uses the `MachineConfig` object to form a secondary bond interface.
++
+[IMPORTANT]
+====
+Do not use the Kubernetes NMState Operator to define or a `NodeNetworkConfigurationPolicy` (NNCP) manifest file to define the secondary interface.
+
+Also ensure that the secondary interface or sub-interfaces when defining a `bond` interface are not used by existing an `br-ex` OVN Kubernetes network deployment.
+====
++
+.. Create the following interface definition files. These files get added to a machine configuration manifest file so that host nodes can access the definition files. 
++
+.Example of the first interface definition file that is named `eno1.config`
+[source,ini]
+----
+[connection]
+id=eno1
+type=ethernet
+interface-name=eno1
+master=bond1
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+----
++
+.Example of the second interface definition file that is named `eno2.config`
+[source,ini]
+----
+[connection]
+id=eno2
+type=ethernet
+interface-name=eno2
+master=bond1
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+----
++
+.Example of the second bond interface definition file that is named `bond1.config`
+[source,ini]
+----
+[connection]
+id=bond1
+type=bond
+interface-name=bond1
+autoconnect=true
+connection.autoconnect-slaves=1
+autoconnect-priority=20
+
+[bond]
+mode=802.3ad
+miimon=100
+xmit_hash_policy="layer3+4"
+
+[ipv4]
+method=auto
+----
++
+.. Convert the definition files to Base64 encoded strings by running the following command:
++
+[source,terminal]
+----
+$ base64 <directory_path>/en01.config
+----
++
+[source,terminal]
+----
+$ base64 <directory_path>/eno2.config
+----
++
+[source,terminal]
+----
+$ base64 <directory_path>/bond1.config
+----
+
+. Prepare the environment variables. Replace `<machine_role>` with the node role, such as `worker`, and replace `<interface_name>` with the name of your additional `br-ex` bridge name.
++
+[source,terminal]
+----
+$ export ROLE=<machine_role>
+----
++
+[source,terminal]
+----
+$ export INTERFACE=<interface_name>
+----
+
+. Create a machine configuration manifest file for configuring the network plugin by entering the following command in your terminal:
```

Comment:
minor layout change: 

This  step `Create a machine configuration manifest file for...` should be moved to line 151 (before the line: `Create a machine configuration manifest file that defines the existing static interface that hosts br-ex1 on any nodes restarted on your cluster`). (we need to create the machine-config yaml, THEN apply it. 

Then the order of operations will be:
- create a manifest yaml to define bond1
- apply said manifest yaml to create bond1
- create a manifest to create the extra bridge on bond1
- apply said manifest file to create extra bridge